### ISSUE TITLE
plot(m, which="knockdown") error at top

### ISSUE THREAD

**Issue by dankelley** [dankelley/mooring#32]
Why the jiggy-jag at the top?

``` r
library(mooring)
m <- mooring(anchor(depth=100), wire(length=90), float("HMB 20"))
md <- discretise(m)
mdk <- knockdown(md, u=0.5)
plot(mdk, which="knockdown")
```

![](https://i.imgur.com/n746y55.png)

<sup>Created on 2021-06-14 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by dankelley** [comment]
It's not just a plotting issue, as shown below.  I'll look into this tomorrow.

``` r
library(mooring)
m <- mooring(anchor(depth=50), wire(length=49), float("HMB 20"))
md <- discretise(m, by=5)
mdk <- knockdown(md, u=0.5)

par(mfrow=c(1,2))

plot(mdk, which="shape", type="p", title="a) mdk shape")
plot(mdk, which="knockdown", type="p", title="b) mdk knockdown")

X <- depth(mdk) - depth(mdk, stagnant=TRUE)
Y <- depth(mdk)
points(X, Y, col=2, cex=2)
```

![](https://i.imgur.com/FjzFyEC.png)

<sup>Created on 2021-06-14 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by dankelley** [comment]
This is the result of some awkward code constructs, and to some extent those result from awkward definitions in the theory.  I will address this by starting with the vignette about the theory (which I'll also rename).

**Comment by dankelley** [comment]
Fixed in "main" branch, comment 0242f90b25ebe351c058cb2613b25e55abeed649, as shown below.

``` r
library(mooring)
m <- mooring(anchor(depth=50), wire(length=49), float("HMB 20"))
md <- discretise(m, by=5)
mdk <- knockdown(md, u=0.5)

par(mfrow=c(1,2))

plot(mdk, which="shape", type="p", title="a) mdk shape")
plot(mdk, which="knockdown", type="p", title="b) mdk knockdown")

X <- depth(mdk) - depth(mdk, stagnant=TRUE)
Y <- depth(mdk)
points(X, Y, col=2, cex=2)
```

![](https://i.imgur.com/3Jo3QC7.png)

<sup>Created on 2021-06-17 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by dankelley** [comment]
Fixed, so closing now.
